package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-delegated-admin",
-  "version": "3.6.4",
+  "version": "3.6.5",
```

Comment:
```suggestion
  "version": "3.7.0",
```

Reply:
> Hi @jhiner,
> 
> When we update dependencies, we usually bump the minor version. Patch automatically gets deployed and there is a risk of failure on running on webtask cloud or appliance if they're not available.

Gotcha, didn't realize that. I opted for the patch because I thought it better to automatically roll this out to customers, but I see the risk.

In that case, I think it makes more sense to update these in the 4.0.0 version instead (WIP here https://github.com/auth0-extensions/auth0-delegated-administration-extension/pull/184/commits/ca7c3135392c678b410cea7b2c8c0f752ef7eb25). I think most customers if upgrading would go to 4.x instead. Unless you disagree, I'll close this PR and instead carry this work into the 4.x PR. Thanks.

Reply:
Ok that makes sense, i see the PR here: https://github.com/auth0-extensions/auth0-delegated-administration-extension/pull/184

Yea, I'd do that.